com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Slots/DiffusionProfileInputMaterialSlot.cs

```diff
@@ -37,7 +37,7 @@ private class DiffusionProfileSerializer
         DiffusionProfileSettings m_DiffusionProfileAsset;
 
         [SerializeField]
-        int m_Version;
+        int m_LocalVersion;
 
```

Comment:
if this name change is supposed to pick up old serializes values, you will need to tell it to convert from the old name